openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/AuthenticationRequestParameters.kt

```diff
@@ -124,6 +133,7 @@ data class AuthenticationRequestParameters(
      * that would normally be provided to an OP during Dynamic RP Registration.
      * It MUST not be present if the RP uses OpenID Federation 1.0 Automatic Registration to pass its metadata.
      */
+    @Deprecated("Removed from OpenId4Vp Draft 21")
```

Comment:
Please remove the comment, and also mark usages of this property with `@Suppress("DEPRECATION")` on method level